plugins/processors/starlark/README.md

```diff
@@ -4,60 +4,69 @@ The `starlark` processor calls a Starlark function for each matched metric,
 allowing for custom programmatic metric processing.
 
 The Starlark language is a dialect of Python, and will be familiar to those who
-have experience with the Python language.  However, keep in mind that it is not
-Python and that there are major syntax [differences][#Python Differences].
+have experience with the Python language. However, there are major [differences](#python-differences).
 Existing Python code is unlikely to work unmodified.  The execution environment
 is sandboxed, and it is not possible to do I/O operations such as reading from
 files or sockets.
 
-The Starlark [specification][] has details about the syntax and available
+The Starlark [specification] has details about the syntax and available
 functions.
 
 Telegraf minimum version: Telegraf 1.15.0
 
 ### Configuration
 
+The Starlark source can be set as a string in this configuration file, or
+by referencing a file containing the script.  Only one source or script
+should be set at once.
+
+
+#### Example
+
 ```toml
 [[processors.starlark]]
-  ## The Starlark source can be set as a string in this configuration file, or
-  ## by referencing a file containing the script.  Only one source or script
-  ## should be set at once.
-  ##
```

Comment:
I would keep the general configuration in there. It's a good reference to have what is the plugin configuration how it is in the `telegraf.conf` as a reference for users to copy & paste from the readme.

Reply:
Okay. Will update—